internal/apiclient/dashboard.go

```diff
@@ -107,20 +118,26 @@ func newRunInput(ctx *config.RunContext, out output.Root) (*runInput, error) {
 	}, nil
 }
 
-func (c *DashboardAPIClient) AddRun(ctx *config.RunContext, out output.Root) (AddRunResponse, error) {
+func (c *DashboardAPIClient) AddRun(ctx *config.RunContext, out output.Root, commentBasicSyntax bool) (AddRunResponse, error) {
 	response := AddRunResponse{}
 
 	ri, err := newRunInput(ctx, out)
 	if err != nil {
 		return response, err
 	}
 
+	commentFormat := "MARKDOWN_HTML"
+	if commentBasicSyntax {
+		commentFormat = "MARKDOWN"
+	}
```

Comment:
If we change the `AddRun` input to accept a `CommentFormat` type rather than a bool this could give us:

1. A closer representation of the graphql enum
2. Means that we could more easily expand options in the future

Code would look something like this (untested):

```suggestion
type CommentFormat string

var (
    CommentFormatMarkdownHTML = "MARKDOWN_HTML"
    CommentFormatMarkdown = "MARKDOWN"
)

func (c *DashboardAPIClient) AddRun(ctx *config.RunContext, out output.Root, commentFormat CommentFormat) (AddRunResponse, error) {
	response := AddRunResponse{}

	ri, err := newRunInput(ctx, out)
	if err != nil {
		return response, err
	}
```

Thoughts?

Reply:
that's fine. what I really would like is a nice way to optionally request the governanceComment.  we only need it for comment and it's potentially expensive for the dashboard to generate.  Maybe i'll fmtF the query string and conditionally set %s = "governanceComment" or ""